tests/unit/store/peek-record-test.js

```diff
@@ -41,3 +41,20 @@ test('peekRecord should return null if the record is not in the store ', functio
     assert.equal(null, store.peekRecord('person', 1), 'peekRecord returns null if the corresponding record is not in the store');
   });
 });
+
+test('peekRecord should assert if not passed both model name and id', function(assert) {
+  run(() => {
+    assert.expectAssertion(() => {
+      store.peekRecord('my-id');
+    }, /You need to pass both a model name and id/);
+  });
+});
+
+test('peekRecord should assert if passed a model class instead of model name', function(assert) {
```

Comment:
@bmac Thanks for pointing that out. I made the change and updated this PR. I'll keep an eye on it to confirm the tests now pass.